packages/wp-story-editor/src/api/media.js

```diff
@@ -169,7 +165,7 @@ export function updateMedia(config, mediaId, data) {
     path: `${config.api.media}${mediaId}/`,
     data,
     method: 'POST',
-  }).then(snakeToCamelCaseObjectKeys);
+  }).then(getResourceFromAttachment);
```

Comment:
As I can see `updateMedia` is being used at these many places, and nowhere have they used its response. So shouldn't we omit the `.then(getResourceFromAttachment);` part?

https://github.com/google/web-stories-wp/blob/acb3df5148d37e8192023c70ea2d9256cf86241d/packages/story-editor/src/app/media/utils/useDetectBaseColor.js#L69-L71
https://github.com/google/web-stories-wp/blob/acb3df5148d37e8192023c70ea2d9256cf86241d/packages/story-editor/src/app/media/utils/useProcessMedia.js#L83-L88
https://github.com/google/web-stories-wp/blob/acb3df5148d37e8192023c70ea2d9256cf86241d/packages/story-editor/src/app/media/utils/useProcessMedia.js#L95-L99
https://github.com/google/web-stories-wp/blob/acb3df5148d37e8192023c70ea2d9256cf86241d/packages/story-editor/src/app/media/utils/useDetectVideoHasAudio.js#L81-L83
https://github.com/google/web-stories-wp/blob/acb3df5148d37e8192023c70ea2d9256cf86241d/packages/story-editor/src/app/media/utils/useUploadVideoFrame.js#L90-L96
https://github.com/google/web-stories-wp/blob/acb3df5148d37e8192023c70ea2d9256cf86241d/packages/story-editor/src/components/library/panes/media/local/mediaEditDialog.js#L121

Reply:
Yeah not sure about it, feels like this might change in the future and we'd need the return value. Seems like something one would expect for updating media